Project-Euler/Problem019.js

```diff
@@ -0,0 +1,38 @@
+/*
+You are given the following information, but you may prefer to do some research for yourself.
+
+  * 1 Jan 1900 was a Monday.
+  * Thirty days has September,
+    April, June and November.
+    All the rest have thirty-one,
+    Saving February alone,
+    Which has twenty-eight, rain or shine.
+    And on leap years, twenty-nine.
+  * A leap year occurs on any year evenly divisible by 4, but not on a century unless it is divisible by 400.
+
+How many Sundays fell on the first of the month during the twentieth century (1 Jan 1901 to 31 Dec 2000)?
+
+*/
+
+export function countingSundays () {
+  let numberOfSundays = 0
+  let dow = 2
+
+  // The number od days in each month of the year
+  const months = [31, 0, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
+
+  for (let y = 1901; y <= 2000; y++) {
```

Comment:
Consider parameterizing the year range (you don't have to though)

Reply:
I think that this way is more legible